modules/programs/rofi.nix

```
@@ -207,6 +207,17 @@ in {
       '';
     };
 
+    plugins = mkOption {
+      default = [ ];
+      type = types.listOf types.package;
+      description = ''
+        List of rofi plugins to be installed.
+      '';
+      example = literalExample ''
+        [ pkgs.rofi-calc ];
+      '';
```

no semicolon necessary in the example text:

```suggestion
      example = literalExample ''[ pkgs.rofi-calc ]'';
```